site/content/docs/main/file-system-backup.md

```diff
@@ -546,7 +546,7 @@ that it's backing up for the volumes to be backed up using FSB.
 3. Velero then creates a `PodVolumeBackup` custom resource per volume listed in the pod annotation  
 4. The main Velero process now waits for the `PodVolumeBackup` resources to complete or fail  
 5. Meanwhile, each `PodVolumeBackup` is handled by the controller on the appropriate node, which:
-    - has a hostPath volume mount of `/var/lib/kubelet/pods` to access the pod volume data
+    - has a hostPath volume mount of `/var/lib/kubelet/pods` and `/var/lib/kubelet/plugins` to access the pod volume data
```

Comment:
File system backup doesn't support block mode, so look like the node-agent doesn't need to mount `/var/lib/kubelet/plugins`?